src/resolvers.js

```diff
@@ -85,3 +85,20 @@ export async function downloadExportedTheme() {
 		parse: false,
 	} );
 }
+
+export async function getFontFamilies() {
+	try {
+		const response = await apiFetch( {
+			path: '/create-block-theme/v1/font-families',
+			method: 'GET',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		} );
+		return response.data;
+	} catch ( e ) {
+		// eslint-disable-next-line no-console
+		console.error( e );
```

Comment:
Remove console statement

```suggestion
		
```

Reply:
handled it here https://github.com/WordPress/create-block-theme/pull/649/commits/18839a3c37b6bff4cae725f81bd3b137c463c882